package.json

```diff
@@ -39,7 +39,6 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "jest": "^27.1.1",
     "jest-dom": "^4.0.0",
-    "moment": "^2.29.1",
```

Comment:
Is moment being used anywhere else?

Reply:
I use it to display the date on each review, but sometimes it needs to reinstall for merging.